### DISCUSSION TITLE
Issue with Cyrillic

### DISCUSSION THREAD

**Discussion by utaliev-damir** [community/community#37387] [Other Feature Feedback, Questions, & Ideas]
Hi Team,

Could you please advise what happened with GitHub, previously all my notebooks (ipynb) that were loaded to git with Cyrillic displayed correctly, now I see this (on screenshot). Also I noticed same problem with other users' notebooks 
![image](https://user-images.githubusercontent.com/111133501/198105477-7721c6d2-9f07-495c-9cc6-39e387a0d0bc.png)

**Comment by airtower-luna** [comment]
I've seen another post about broken characters today (#37324), so it's probably a bug. I recommend you [send a bug report](https://support.github.com/contact/bug-report) to GitHub support.

**Reply by utaliev-damir** [reply]
thank you! it seems sort of update happening on git.
